src/content/docs/fr/guides/migrate-to-astro/from-gatsby.mdx

```diff
@@ -0,0 +1,585 @@
+            ? 
+              <h1 class="main-heading">
+              <a href="/">{title}</a>
+              </h1>
+            : 
+              <h1 class="main-heading">
+              <a class="header-link-home" href="/">Accueil</a>
+              </h1>
+            }  
+          </header>
+          <main>
+            <slot />
+          </main>
+          <footer>
+            &#169; {new Date().getFullYear()}, Construit avec
+            {` `}
+            <a href="https://www.astro.build">Astro</a>
+          </footer>
+        </div>
+      </body>
+    </html>
+    ```
+
+5. Mettez à jour `index.astro` pour utiliser cette nouvelle mise en page et lui transmettre les propriétés `title` et `pathname` nécessaires :
+
+    ```astro title="src/pages/index.astro"
+    ---
+    import Layout from '../layouts/Layout.astro';
+    const pagePathname = Astro.url.pathname
+    ---
+    <Layout title="Page d'accueil" pathname={pagePathname}>
+        <p>Astro</p>
+    </Layout>
+    ```
+    :::tip
+    Vous pouvez [récupérer le chemin de la page actuelle en utilisant `Astro.url`](/fr/reference/api-reference/#astrourl).
+    :::
+
+6. Pour tester l'en-tête conditionnel, créez une deuxième page, `about.astro` en utilisant le même modèle :
+
+    ```astro title="src/pages/about.astro"
+    ---
+    import Layout from '../layouts/Layout.astro';
+    const pagePathname = Astro.url.pathname
+    ---
+    <Layout title="À propos" pathname={pagePathname}>
+        <p>À propos</p>
+    </Layout>
+    ```
+
+    Vous devriez voir un lien vers « Accueil » uniquement lorsque vous visitez la page À propos.
+</Steps>
+
+## Ressources communautaires
+
+- Article de blog : [Migration de Gatsby à Astro](https://loige.co/migrating-from-gatsby-to-astro/) (en).
+
+- Article de blog : [La migration vers Astro était facile](https://joelhooks.com/migrating-to-astro-was-ez) (en).
+
+- Article de blog : [Mon passage de Gatsby à Astro](https://www.joshfinnie.com/blog/my-switch-from-gatsby-to-astro/) (en).
+
+- Article de blog : [Pourquoi je suis passé de Gatsby à Astro](https://dev.to/askrodney/why-i-moved-to-astro-from-gatsby-3fck) (en).
+
+- Article de blog : [Migrer mon site web de Gatsby vers Astro](https://dev.to/flashblaze/migrating-my-website-from-gatsby-to-astro-2ej5) (en).
+
+- Article de blog : [Une autre migration : de Gatsby à Astro](https://logarithmicspirals.com/blog/migrating-from-gatsby-to-astro/) (en).
+
+- Article de blog : [Pourquoi et comment j'ai déplacé mon blog de Gatsby et React vers Astro et Preact](https://www.helmerdavila.com/blog/en/why-and-how-i-moved-my-blog-away-from-gatsby-and-react-to-astro-js-and-preact) (en).
+
+:::note[Vous avez une ressource à partager ?]
+Si vous trouvez (ou créez !) une vidéo ou un article de blog utile sur la conversion d'un site Gatsby vers Astro, [éditez cette page](https://github.com/withastro/docs/edit/main/src/content/docs/en/guides/migrate-to-astro/from-gatsby.mdx) et ajoutez-le ici !
```

Comment:
```suggestion
Si vous trouvez (ou créez !) une vidéo ou un article de blog utile sur la conversion d'un site Gatsby vers Astro, [éditez cette page](https://github.com/withastro/docs/edit/main/src/content/docs/fr/guides/migrate-to-astro/from-gatsby.mdx) et ajoutez-le ici !
```

Reply:
@thomasbnt This one I hesitated but the contribution guide explains that all changes must be done on the English pages. Then we update the translations, not the inverse. What do you think? Should I update?

Reply:
> @thomasbnt This one I hesitated but the contribution guide explains that all changes must be done on the English pages. Then we update the translations, not the inverse. What do you think? Should I update?

oooh yeah, you right! sorry!